### DISCUSSION TITLE
Recommendation on empty `aria-label` attributes

### DISCUSSION THREAD

**Discussion by danielchalmers** [nvaccess/nvda#16651] [Q&A]
Hi, I have a text property binding for an `aria-label` in Blazor. If the property value is `Hello` then `aria-label=Hello`.

Is it a concern if the property is null or empty and the DOM has an empty `aria-label` with no value? How is that handled? Thanks.

**Comment by Adriani90** [comment]
Do you have a minimal test case in Codepen to see this in practice? Otherwise it is hard to say.

**Reply by danielchalmers** [reply]
Simply

```html
<div aria-label>
</div>
```

**Comment by Adriani90** [comment]
This will probably not have any negative influence on screen reader's behavior, at least speaking for NVDA. It would be good to see a minimal test case on codepen like a minimal webpage or something like that. From the code snipped you set it doesn't seem to have any negative effect.
